google-map-directions.html

```diff
@@ -71,7 +78,7 @@
        * @type string|google.maps.LatLng
```

Comment:
I am going to wait on this until I upgrade the doc page to iron-doc. Hydrolysis has lots of new goodies, I think generating types automatically is one of them.